pyls/config.py

```diff
@@ -11,8 +12,8 @@
 
 class Config(object):
 
-    def __init__(self, root_path, init_opts):
-        self._root_path = root_path
+    def __init__(self, root_uri, init_opts):
+        self._root_path = unquote(urlparse(root_uri).path)
```

Comment:
Sorry, I meant to just store a `self._root_uri` and expose a `root_uri` property. Breaking the API of the Config object, but it doesn't really matter.